skimage/measure/_regionprops.py

```diff
@@ -166,6 +167,23 @@ def extent(self):
         return self.moments[0, 0] / (rows * cols)
 
     @property
+    def feret_diameter(self):
+        from ..morphology.convex_hull import convex_hull_image
+        from ._find_contours import find_contours
+        from scipy.spatial.distance import pdist
```

Comment:
Moving `from ..morphology.convex_hull import convex_hull_image` to the top of the module results in an import loop which seems to require that both modules be compiled before they can be imported, resulting in a deadlock. I could try separating out the offending loop in the `morphology` module. What do you recommend?